optional_plugins/runner_remote/avocado_runner_remote/__init__.py

```diff
@@ -433,12 +433,14 @@ def run_test(self, references, timeout):
         :return: a dictionary with test results.
         """
         extra_params = []
-        mux_files = getattr(self.job.args, 'mux_yaml', [])
-        if mux_files:
-            extra_params.append("-m %s" % " ".join(mux_files))
+        for arg in ["--mux-yaml", "--dry-run"]:
+            key = arg[2:].replace('-', '_')
```

Comment:
Options *may* use  `dest` with a different value, which would break this, but, since we're using a whitelist and these options don't do that, then we're fine.